src/components/presentationals/AuthenticationCard.jsx

```diff
@@ -117,4 +80,10 @@ const title = {
   class: 'text-center',
 };
 
+const loginButtonValues = {
```

Comment:
We don't need to create this constant here, we should apply these props to the component directly.

```javascript 
<Button type="submit" value="CONTINUE" class="btn-dark" />

Reply:
We have refactored it to look like this now